lib/std/Target.zig

```diff
@@ -1782,6 +1782,8 @@ pub const DynamicLinker = struct {
             .visionos,
             => init("/usr/lib/dyld"),
 
+            .serenity => init("/usr/lib/Loader.so"),
+
             // Operating systems in this list have been verified as not having a standard
             // dynamic linker path.
```

Comment:
I'm not sure who verified this but it's not true :)

https://github.com/SerenityOS/serenity/blob/8a7d9a7b68c76f9f249c3ec7f2f0bb2e6ca0ed6e/Userland/Libraries/LibCore/System.cpp#L187